regression/esbmc-cpp11/README.md

```diff
@@ -0,0 +1,16 @@
+Benchmark Description
+
+| Test Case| Main Features Tested |
+| :----:   | :---                 |
```

Comment:
This line seems strange

Reply:
They are for markdown table text alignments.